docs/extending-overseerr/fail2ban.md

```diff
@@ -0,0 +1,93 @@
+# Protecting Your Server Using Fail2ban
+
```

Comment:
```suggestion

{% hint style="danger" %}
If you are running Overseerr in a Docker container, make sure that the `PROXY` environment variable is set to `yes`.
{% endhint %}

```

Reply:
Was about to add this one but you beat me to it while I was having a shower :p

Reply:
Can you first change this one from danger to warning, please